redisinsight/ui/src/slices/rdi/pipeline.ts

```diff
@@ -72,6 +90,42 @@ const rdiPipelineSlice = createSlice({
     deployPipelineFailure: (state) => {
       state.loading = false
     },
+    stopPipeline: (state) => {
+      state.stop.loading = true
+      state.stop.error = ''
+    },
+    stopPipelineSuccess: (state) => {
+      state.stop.loading = false
+      state.stop.error = ''
+    },
+    stopPipelineFailure: (state, { payload }: PayloadAction<string>) => {
+      state.stop.loading = false
+      state.stop.error = payload
+    },
+    startPipeline: (state) => {
+      state.start.loading = true
+      state.start.error = ''
+    },
+    startPipelineSuccess: (state) => {
+      state.start.loading = false
+      state.start.error = ''
+    },
+    startPipelineFailure: (state, { payload }: PayloadAction<string>) => {
+      state.start.loading = false
+      state.start.error = payload
+    },
+    resetPipeline: (state) => {
+      state.reset.loading = true
+      state.reset.error = ''
+    },
+    resetPipelineSuccess: (state) => {
+      state.reset.loading = false
+      state.reset.error = ''
+    },
+    resetPipelineFailure: (state, { payload }: PayloadAction<string>) => {
+      state.reset.loading = false
+      state.reset.error = payload
+    },
```

Comment:
a lot similar actions, could you think how to have one? Maybe add some variables (isStarted)? Do we use all these errors at the same time?